projects/ngrx.io/content/guide/store/testing.md

```diff
@@ -59,7 +61,126 @@ describe('Auth Guard', () => {
 });
 </code-example>
 
-### Using Store for Integration Testing
+### Example: Effect injecting state
+
+The mock store can simplify testing Effects that inject State using the RxJs `withLatestFrom` operator.  The example below shows the `addBookToCollectionSuccess$` effect displaying a different alert depending on the number of books in the collection state.
+
+<code-example header="collection.effects.ts">
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType, createEffect } from '@ngrx/effects';
+import { map, withLatestFrom } from 'rxjs/operators';
+import { CollectionApiActions } from '@example-app/books/actions';
+import * as fromBooks from '@example-app/books/reducers';
+import { Store, select } from '@ngrx/store';
```

Comment:
I think since this is a documentation resource it should be helpful if we group all the lib imports and then import the example app imports.

Reply:
Agreed, reorganized in 18e4a12.